scripts/deploy_infrastructure.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ENV=$1
+TERRAFORM_STATE_BUCKET="${TERRAFORM_STATE_BUCKET:-census-eq-terraform-tfstate}"
+
+terraform init --upgrade --backend-config prefix=${ENV} --backend-config bucket=${TERRAFORM_STATE_BUCKET}
+
+if [ ! -z $EXISTING_PROJECT_ID ]; then
+    echo "Using existing project_id: $EXISTING_PROJECT_ID"
+
+    if terraform state list | grep -q "google_project.project"; then
+        echo "State contains a google project. Not importing"
+    else
+        echo "State does not contain a google project. Importing $EXISTING_PROJECT_ID"
+        terraform import google_project.project $EXISTING_PROJECT_ID
+    fi
+fi
+
+# Roll out infrastructure
+terraform apply -auto-approve -var 'env=${ENV}'
+
+PROJECT_ID=$(terraform output google_project_id)
```

Comment:
I get an error here:

```
++ terraform output google_project_id
The output variable requested could not be found in the state
file. If you recently added this to your configuration, be
sure to run `terraform apply`, since the state won't be updated
with new output variables until that command is run.
+ PROJECT_ID=
```